website/src/layouts/OrganismPage/OrganismPageLayout.astro

```diff
@@ -23,6 +27,7 @@ const lastUpdatedDate = await getLastUpdatedDate(getLapisUrl(view.organismConsta
     </div>
     <slot />
     <div class='flex justify-center' slot='secondary-footer'>
```

Comment:
I think the alignment is lightly off on mobile:
![grafik](https://github.com/user-attachments/assets/6f8ea30a-dbcf-4ddb-b8b9-12abd16b70b8)
Maybe a combination of max width and justify-between might help. We could also get rid of the "mr-4" through a "gap-x".

Reply:
Stacking is a great idea. I think it looks good now and the point before the sm breakpoint is not that bad.